Sources/AudioKit/Nodes/Playback/AudioPlayer/AudioPlayer.swift

```diff
@@ -269,6 +269,19 @@ public class AudioPlayer: Node {
 }
 
 extension AudioPlayer: HasInternalConnections {
+    /// Check if the playerNode is already connected to the mixerNode
+    func playerNodeConnectedToMixerNode(engine: AVAudioEngine) -> Bool {
```

Comment:
Can we prefix that function with "is" to be "isPlayerNodeConnectedToMixerNode" to really drive home that its a boolean? Also (and I didn't check yet), does the engine need to be sent to this? Probably can get the engine from the node and this can be a computed variable instead of a function.